src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -9,28 +9,14 @@
 import com.mojang.serialization.Codec;
 import com.mojang.serialization.Lifecycle;
 import com.mojang.serialization.codecs.RecordCodecBuilder;
-import java.util.Arrays;
-import java.util.EnumSet;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
-import java.util.concurrent.CompletableFuture;
-import java.util.function.Consumer;
-import java.util.function.Function;
-import java.util.function.Predicate;
 import net.minecraft.DetectedVersion;
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.renderer.BiomeColors;
 import net.minecraft.commands.Commands;
 import net.minecraft.commands.synchronization.ArgumentTypeInfo;
 import net.minecraft.commands.synchronization.ArgumentTypeInfos;
 import net.minecraft.commands.synchronization.SingletonArgumentInfo;
-import net.minecraft.core.BlockPos;
-import net.minecraft.core.Holder;
-import net.minecraft.core.HolderLookup;
-import net.minecraft.core.MappedRegistry;
-import net.minecraft.core.RegistryCodecs;
+import net.minecraft.core.*;
```

Comment:
Use `gradlew applySpotless` to undo all these changes.

Reply:
applySpotless isn't a thing it looks like:
![image](https://github.com/neoforged/NeoForge/assets/84230130/929cb9b0-6a42-4f27-881d-91c8d512a2de)

Reply:
right, it's `spotlessApply`

Reply:
it's just complaining rather than fixing it
![image](https://github.com/neoforged/NeoForge/assets/84230130/c250ba2b-de4c-43b2-8b28-f501c758ea94)

Reply:
yeah, if you scroll up the stack trace you'll see that it can't fix some errors (e.g. wildcard imports must be fixed manually).

Reply:
ok, thanks!